src/mineflayerTest.js

```diff
@@ -8,7 +8,7 @@ let testList = []
 let currentDir = ''
```

Comment:
Looks like the test is failing because this variable is unused. It can be safely removed.

Reply:
fixed